src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,49 @@
 
 class Animal {
   // write your code here
+  static alive = [];
```

Comment:
You can try to create it outside of the class to avoid eslint error
`Animal.alive = []`